XCUITests/CronTabsPerfTests.swift

```diff
@@ -0,0 +1,243 @@
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // activity measurement here
+            }
+        }
+    }
+
+    func testPerfTabs320() {
+        if #available(iOS 13.0, *) {
+            app.launch()
+            measure(metrics: [
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // activity measurement here
+            }
+        }
+    }
+
+    func testPerfTabs640() {
+        if #available(iOS 13.0, *) {
+            app.launch()
+            measure(metrics: [
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // activity measurement here
+            }
+        }
+    }
+
+    func testPerfTabs1280() {
+        if #available(iOS 13.0, *) {
+            app.launch()
+            measure(metrics: [
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // activity measurement here
+            }
+        }
+    }
+    
+    // Additional Scenarios
+    // 1. Start app
+    // 2. Open tabs Tray
+    // 3. Open 1 new tab
+    // 4. Close 1 tab
+    // 5. Cloase all tabs
+    // 6. Switch to private browsing
+    // 7. Switch back from private browsing
+    // 8. Create new tab in private browsing
+
+    func testPerfTabs20tabTray() {
+
+        if #available(iOS 13.0, *) {
+            app.launch()
+            measure(metrics: [
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // go to tab tray
+                //navigator.performAction(Action.TogglePrivateMode)
+                navigator.goto(TabTray)
+
+            }
+        }
+    }
+    
+    func testPerfTabs1280tabTray() {
+
+        if #available(iOS 13.0, *) {
+            app.launch()
+            measure(metrics: [
+                XCTClockMetric(), // to measure timeClock Mon
+                XCTCPUMetric(), // to measure cpu cycles
+                XCTStorageMetric(), // to measure storage consuming
+                XCTMemoryMetric()]) {
+                // go to tab tray
+                //navigator.performAction(Action.TogglePrivateMode)
+                navigator.goto(TabTray)
+
+            }
+        }
+    }
+
+    func testTabs5Setup() {
```

Comment:
I would add a comment above this test to specify what it does and would leave it as commented since it will not be part of the performance test run... Also the commented lines not needed in the test could be removed